networking/hardware_networks/using-dpdk-and-rdma.adoc

```diff
@@ -28,6 +39,13 @@ include::modules/nw-sriov-rdma-example-mellanox.adoc[tag=content]
 [id="additional-resources_using-dpdk-and-rdma"]
 == Additional resources
 
+* xref:../../scalability_and_performance/cnf-create-performance-profiles.adoc#cnf-about-the-profile-creator-tool_cnf-create-performance-profiles[Creating a performance profile].
+* xref:../../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#adjusting-nic-queues-with-the-performance-profile_cnf-master[Reducing NIC queues using the Node Tuning Operator].
+* xref:../../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#performance-addon-operator-provisioning-worker-with-real-time-capabilities_cnf-master[Provisioning a worker with real-time capabilities].
```

Comment:
This xref also does not work after a rebase, because it uses the renamed assembly that I mentioned in my preceding comment. I can see the section in the renamed page, so I can provide the updated xref that is required here:

````
* xref:../../scalability_and_performance/cnf-low-latency-tuning.adoc#performance-addon-operator-provisioning-worker-with-real-time-capabilities_cnf-master[Provisioning a worker with real-time capabilities]
````